packages/nc-gui/components/virtual-cell/Formula.vue

```diff
@@ -17,16 +17,16 @@ const urls = computed(() => replaceUrlsWithLink(result.value))
 const timeout = 3000 // in ms
 
 const showEditFormulaWarning = refAutoReset(false, timeout)
+
 const showClearFormulaWarning = refAutoReset(false, timeout)
 
 useSelectedCellKeyupListener(inject(ActiveCellInj, ref(false)), (e: KeyboardEvent) => {
   switch (e.key) {
     case 'Enter':
       showEditFormulaWarning.value = true
       break
-    case 'Delete':
+    default:
       showClearFormulaWarning.value = true
-      break
   }
```

Comment:
since we are setting it to true for any keypress switch case is not at all required here.